pkg/kp/kv.go

```diff
@@ -171,3 +222,16 @@ func (c consulStore) Ping() error {
 	}
 	return nil
 }
+
+func addTimeStamp(value WatchResult) (time.Time, WatchResult) {
+	currentTime := time.Now()
+	if value.Time == "" {
+		value.Time = currentTime.String()
+	}
+
+	return currentTime, value
+}
+
+func healthPath(service, node string) string {
+	return fmt.Sprintf("%s%s/%s", service, "health", node)
```

Comment:
should there be a slash between the first two `%s`? If not, then this implies service should end with a slash?